### DISCUSSION TITLE
spacy annotations!

### DISCUSSION THREAD

**Discussion by oktrained** [explosion/spaCy#13179] [Help: Best practices]
Hello,

I am working with legal documents, from which I primarily want to extract entities that involve monetary mentions. There are various entities with monetary values, and I have annotated them to train a named entity recognition (NER) model. However, the model is not performing well, and I am uncertain about the reasons. It could be due to some entities being annotated over long spans, such as 5 to 6 sentences. I annotated them for longer spans because sometimes the key information for the entity I want to extract is not confined to a single sentence. If anyone who has worked with similar documents could provide assistance, it would be greatly appreciated. Thank you.

**Comment by rmitsch** [comment]
Hi @HarshadDolas07!

> It could be due to some entities being annotated over long spans, such as 5 to 6 sentences. I annotated them for longer spans because sometimes the key information for the entity I want to extract is not confined to a single sentence.

Could you elaborate on this and give some examples?
